backend/audit/test_etl.py

```diff
@@ -374,15 +397,14 @@ def test_load_all_with_errors_1(self):
             corrective_action_plan=self._fake_corrective_action_plan(),
             secondary_auditors=self._fake_secondary_auditors(),
             additional_ueis=self._fake_additional_ueis(),
-            # audit_information=self._fake_audit_information(),  # TODO: Uncomment when SingleAuditChecklist adds audit_information
```

Comment:
Any reason why this is removed here but used [here](https://github.com/GSA-TTS/FAC/pull/1912/files#diff-b594cf1258aab056b2cf43b2bfeec18a7c58b02dd6c0455c4f2683b1aad4cc6dR372) ?